enterprise/releases/2023-07-0.mdx

```diff
@@ -0,0 +1,80 @@
+    - Previously, if autovacuum logs included such information, the collector failed to match the log line and the events would not be classified correctly in Log Insights
+  - In case you have installed the collector separately, we recommend upgrading the collector package/container and reviewing the [full changelog](https://github.com/pganalyze/collector/blob/main/CHANGELOG.md#0501------2023-06-29).
+
+
+## Performance
+
+* Snapshot processing
+  - Improve speed of lock graph analysis for blocking queries alert <!-- PR 2987 -->
+  - Re-introduce limit of length on query parsing for very large queries (more than 200,000 characters) <!-- PR 2956 -->
+    - After the removal of the old 20,000 limit in the prior Enterprise release (v2023.06.0) we've received reports of this causing crashes
+    - In our testing the problem case here is when queries are extremely long (sometimes query texts exceeding 1MB in length) - this release re-introduces a limit of 200,000 (10x the old limit), which allows most cases but avoids the extreme cases that cause the snapshot worker to crash
+    - Queries exceeding this length limit will not be considered for Index Advisor, and will show under the "Other" statement type in the Query Performance view
+  - Adjust estimation method used for calculate overview percentile metrics to improve snapshot processing speed <!-- PR 2950 -->
+    - This provides a significant speedup for snapshot processing in some edge cases (which may have previously timed out)
+    - Note that you may see small differences in top-level percentile numbers going forward compared to historic calculations
+
+
+## Bugfixes
+
+* Config Settings
+  - Move autovacuum_vacuum_insert_threshold and autovacuum_vacuum_insert_scale_factor from Other Settings config group to Autovacuum config group <!-- PR 2905 -->
+  - Update and sync Postgres setting config group with Postgres 15 <!-- PR 2917 -->
+* Query and table overview: Fix edge case that accidentally counted the same timeseries data point twice <!-- PR 2962 -->
+  - This edge case only occurred when viewing a very short time frame (2 hours or less), within certain conditions. If the error occurred, you might have seen the "% of Total Runtime" sum up to more than 100%, or other summary statistics to be higher than what you would get when viewing the detail statistics individually.
+* Index Advisor
+  - Fix incorrect method/costs shown for scans on query details <!-- PR 2959 -->
+    - This bug caused the scan list to pull in costs and methods from other queries in the same database on the same table, that had an identical scan (i.e. same set of WHERE/JOIN conditions)
+    - Whilst this information was not utilized for index recommendations, it may have been misleading when reading the scan information on the query details page.
+* Alerts & Check-Up
+  * Run each daily check in a separate background jobs for each server <!-- PR 2907 -->
```

Comment:
This feels like better fit in performance or features category (but also it doesn't exactly fit, so I'm also fine to leave it here)

Reply:
Yeah, I feel like this would fit in a "Stability" category if we had it - I'll put it in "Features" for now.